cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -400,6 +400,7 @@ ExternalProject_Add(
     ${b2_cmd}
     ${verbose_output}
     ${build_opts}
+    --ignore-site-config # Ignore Gentoo specific optimization "none" in site config that only the patched bjam of Gentoo can understand.
```

Comment:
Since build scheme of the packages changed we need to update `PACKAGE_INTERNAL_DEPS_ID` of Boost.

Example:
* https://github.com/ruslo/hunter/pull/863/commits/bf3c5f7d122067f0d9a1815c0f808c258acd3dbc

Explanation:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html